hugo/content/blog/2019-08-29-mumble-1-3-0/index.html

```diff
@@ -0,0 +1,1284 @@
+</div>
+
+
+<div class="item">
+    <h3 class="item-title">[User][PositionalAudio] GTA V</h3>
+    <div class="item-desc"></div>
+    <div class="item-prs">Pull Requests: <a class="pr-link" href="https://github.com/mumble-voip/mumble/pull/2310">#2310</a><a class="pr-link" href="https://github.com/mumble-voip/mumble/pull/2315">#2315</a><a class="pr-link" href="https://github.com/mumble-voip/mumble/pull/2345">#2345</a><a class="pr-link" href="https://github.com/mumble-voip/mumble/pull/2384">#2384</a><a class="pr-link" href="https://github.com/mumble-voip/mumble/pull/2423">#2423</a><a class="pr-link" href="https://github.com/mumble-voip/mumble/pull/2953">#2953</a><a class="pr-link" href="https://github.com/mumble-voip/mumble/pull/2977">#2977</a></div>
+    <div class="item-commits">Commits: <details><summary><a class="commit-hashlink" href="https://github.com/mumble-voip/mumble/commit/f38363f2" title="Merge PR #2310: Positional Audio support for Grand Theft Auto V">f38363f2</a><a class="commit-hashlink" href="https://github.com/mumble-voip/mumble/commit/44ea8c86" title="Merge PR #2315: Add GTA V plugin to the build and to the installer.">44ea8c86</a><a class="commit-hashlink" href="https://github.com/mumble-voip/mumble/commit/f4ca0cf2" title="Merge PR #2345: plugins/gtav: Plugin update for game&#39;s latest version">f4ca0cf2</a><a class="commit-hashlink" href="https://github.com/mumble-voip/mumble/commit/046e0ca2" title="Merge PR #2384: plugins/gtav: Support for retail version">046e0ca2</a><a class="commit-hashlink" href="https://github.com/mumble-voip/mumble/commit/48ad19e4" title="Merge PR #2423: plugins/gtav: update plugin to work with version 1.35">48ad19e4</a><a class="commit-hashlink" href="https://github.com/mumble-voip/mumble/commit/d7b26dd9" title="plugins/gtav: update game_name memory addresses">d7b26dd9</a><a class="commit-hashlink" href="https://github.com/mumble-voip/mumble/commit/501651b1" title="plugins/gtav: update plugin to work with version 1.37 (Steam only)">501651b1</a><a class="commit-hashlink" href="https://github.com/mumble-voip/mumble/commit/69defe51" title="Merge PR #2953: plugins/gtav: update plugin to work with version 1.38 (Steam)">69defe51</a><a class="commit-hashlink" href="https://github.com/mumble-voip/mumble/commit
```

Comment:
Typo: `[Infrastructire]` -> `[Infrastructure]`

Reply:
Hah, there is more. I used a spell checker to reveal some more. ;-)